sdk/storage/azure-storage-internal-avro/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 12.7.0-beta.1 (Unreleased)
 
 ### Features Added
+- Added support for 2022-11-02 service version.
```

Comment:
Do we actually release the Avro package?

Reply:
It does get released as part of our GA releases.